modules/doctor/heads.go

```diff
@@ -49,7 +49,7 @@ func synchronizeRepoHeads(ctx context.Context, logger log.Logger, autofix bool)
 		}
 
 		// otherwise, let's try fixing HEAD
-		err := git.NewCommand(ctx, "symbolic-ref").AddDashesAndList("HEAD", repo.DefaultBranch).Run(runOpts)
+		err := git.NewCommand(ctx, "symbolic-ref").AddDashesAndList("HEAD", "refs/heads/"+repo.DefaultBranch).Run(runOpts)
```

Comment:
IMO there is  already a const:

```
// BranchPrefix base dir of the branch information file store on git
const BranchPrefix = "refs/heads/"
```

I would prefer to  use it  instead of hard-coded strings.